bin/mixer_codegen.sh

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env bash
+
+WD=$(dirname $0)
+WD=$(cd $WD; pwd)
+ROOT=$(dirname $WD)
+
+set -e
+
+outdir=$ROOT
+file=$ROOT
+protoc="protoc-min-version -version=3.5.0"
+optimport=$ROOT
+
+while getopts 'f:o:p:i:' flag; do
+  case "${flag}" in
+    f) file+="/${OPTARG}" ;;
+    o) outdir="${OPTARG}" ;;
+    p) protoc="${OPTARG}" ;;
+    i) optimport+=/"${OPTARG}" ;;
+    *) error "Unexpected option ${flag}" ;;
+  esac
+done
+
+# echo "outdir: ${outdir}"
+
+# Ensure expected GOPATH setup
+if [ $ROOT != "${GOPATH-$HOME/go}/src/istio.io/istio" ]; then
+       echo "Istio not found in GOPATH/src/istio.io/"
+       exit 1
+fi
+
+if [ ! -e $GOPATH/bin/protoc-gen-gogoslick ]; then
+echo "Installing protoc-gen-gogoslick..."
+pushd $ROOT
+go install "./vendor/github.com/gogo/protobuf/protoc-gen-gogoslick"
```

Comment:
I think we had several complaints about installing go packages into system bin. Do you think a `go build -o` to a local directory is less invasive?

Reply:
Perhaps. Let's optimize after we get it working.